niworkflows/data/derivatives.json

```diff
@@ -0,0 +1,151 @@
+{
+  "name": "derivatives",
```

Comment:
Now, all PyBIDS processing is centralized in this config. The `niworkflows/reports/figures.json` has been dropped and removed.

Reply:
does "nipreps" work out?